libs/backtesting/PalStrategy.h

```diff
@@ -802,7 +893,7 @@ namespace mkc_timeseries
 					  fillPrice, targetAsPercent);
 	    this->ExitShortAllUnitsAtStop(aSecurity->getSymbol(), processingDate,
 					  fillPrice, stopAsPercent);
-	    instrPos.setRMultipleStop (ShortStopLoss<Decimal> (fillPrice, stopAsPercent).getStopLoss());
+	    const_cast<InstrumentPosition<Decimal>&>(instrPos).setRMultipleStop (ShortStopLoss<Decimal> (fillPrice, stopAsPercent).getStopLoss());
```

Comment:
Hum, not too sure about the `const_cast`
Usually if a member is safely mutable when the object is `const`, we can simply mark that field as `mutable`. It's usually cleaner & helps with reusability

Reply:
@noboruma I fixed the const_cast issue. Have a look.